snuba/perf.py

```diff
@@ -57,14 +57,14 @@ def get_messages(events_file):
     return messages
 
 
-def run(events_file, clickhouse, dataset, repeat=1,
+def run(events_file, dataset, repeat=1,
         profile_process=False, profile_write=False):
     from snuba.consumer import ConsumerWorker
-    from snuba.writer import NativeDriverBatchWriter
+    from snuba.clickhouse import ClickhousePool
+
+    ClickhousePool().execute(dataset.get_schema().get_local_table_definition())
```

Comment:
This is kind of clunky — this should probably be moved to a `create_table` method on `DataSet` or similar in the future, but that's a design decision I didn't want to have to make to solely implement this limited use case.

Reply:
(Specifically, I'd like to avoid exposing something like `get_pool` as public API on `DataSet` if possible because that will couple us to the specific driver API.)